package/yast2-auth-server.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Apr  8 12:12:45 UTC 2016 - hguo@suse.com
+
+- Adept to the latest changes in Tumbleweed to load essential
```

Comment:
- "ad**a**pt"
- "latest changes in Tubmbleweed" is a bit vague. Any specific version of a package that we depend on?
- a Bugzilla reference is missing